code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -3,14 +3,15 @@ GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /
 /proc/meta_gas_list()
 	. = subtypesof(/datum/gas)
 	for(var/gas_path in .)
-		var/list/gas_info = new(6)
+		var/list/gas_info = new(7)
 		var/datum/gas/gas = gas_path
 
 		gas_info[META_GAS_SPECIFIC_HEAT] = initial(gas.specific_heat)
 		gas_info[META_GAS_NAME] = initial(gas.name)
 		gas_info[META_GAS_MOLES_VISIBLE] = initial(gas.moles_visible)
 		if(initial(gas.moles_visible) != null)
 			gas_info[META_GAS_OVERLAY] = new /obj/effect/overlay/gas(initial(gas.gas_overlay))
+		gas_info[META_GAS_FUSION_POWER] = initial(gas.dangerous)
```

Comment:
wait shouldnt this be
 ```DM
initial(gas.fusion_power)
```
?

Reply:
i get this is already merged but that seems like kind of a large oversight dont you  think

Reply:
Yes. Yes it does.